pyproject.toml

```diff
@@ -49,6 +49,7 @@ dev = [
     "pre-commit>=3.3.3",
     "pytest>=7.4.0",
     "pytest-cov>=4.1.0",
+    "pydoclint",
```

Comment:
Why does it appear here? `pre-commit` is run isolated, and it does not seem to be used anywhere in the docs build / GH Actions.

Reply:
IMO it's useful to just be able to run those tools in isolation (e.g., as part of an editor/IDE integration).